Code/Tools/ProjectManager/Source/ProjectButtonWidget.cpp

```diff
@@ -259,15 +261,39 @@ namespace O3DE::ProjectManager
         }
 
         projectActionButton->setText(text);
+        projectActionButton->setMenu(nullptr);
         m_actionButtonConnection = connect(projectActionButton, &QPushButton::clicked, lambda);
     }
 
-    void ProjectButton::SetProjectBuildButtonAction()
+    void ProjectButton::ShowDefaultBuildButton()
     {
-        m_projectImageLabel->GetWarningLabel()->setText(tr("Building project required."));
-        m_projectImageLabel->GetWarningIcon()->setVisible(true);
-        m_projectImageLabel->GetWarningLabel()->setVisible(true);
-        SetProjectButtonAction(tr("Build Project"), [this]() { emit BuildProject(m_projectInfo); });
+        QPushButton* projectActionButton = m_projectImageLabel->GetActionButton();
+        projectActionButton->setVisible(true);
+        projectActionButton->setText(tr("Build Project"));
+        disconnect(m_actionButtonConnection);
+
+        QMenu* menu = new QMenu(this);
+        QAction* autoBuildAction = menu->addAction(tr("Auto-build"));
+        connect( autoBuildAction, &QAction::triggered, this, [this](){ emit BuildProject(m_projectInfo); });
+
+        QAction* openCMakeAction = menu->addAction(tr("Open CMake..."));
+        connect( openCMakeAction, &QAction::triggered, this, [this](){ emit OpenCMake(m_projectInfo); });
```

Comment:
some methods were called "OpenCMakeGUI" but this one is named "OpenCMake", suggest naming them all "OpenCMakeGUI"